omnibus/files/private-chef-cookbooks/private-chef/recipes/plugin_discovery.rb

```diff
@@ -15,8 +15,8 @@
 # limitations under the License.
 #
 
-BACKUP_PLUGIN_LOCATIONS = %w(/opt/opscode/chef-server-plugin.rb
-                             /opt/chef-ha/chef-server-plugin.rb)
+# TODO Does this go away with HA?
```

Comment:
I think it was also used by

- https://github.com/chef/omnibus-ha

which I think is also deprecated for a while now.  

The only other thing I can think of that *may* have been using it was the marketplace/opsworks stuff.  Maybe @ryancragun or @danielsdeleo can confirm that it isn't used there.

Reply:
I'm going to take this on in a separate PR.

Reply:
This is taken up in #1673